datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SetRangerCloudIdentityMappingRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.sdx.api.model;
+
+import java.util.Map;
+
+public class SetRangerCloudIdentityMappingRequest {
```

Comment:
I think some annotations are missing here, like ignore unknown etc

Reply:
For now both of these fields are required. I think once we add support for other cloud providers we might want to loosen that requirement.

Reply:
Thanks! (I don't think I saw similar annotations in the other sdx requets)